### ISSUE TITLE
Cicerone - accessibility

### ISSUE THREAD

**Issue by humoroussmile** [CSGJusticeCenter/MCLCShiny#63]
It appears there may not be built-in accessibility for cicerone when it overlays the app - can you look into this? Explicitly:

1. I cannot keyboard navigate to the "more info" button at the bottom right of the screen - it is only accessible by mouse
2. a screen reader is not reading through the cicerone overlays completely (using NVDA)
    a. it does read through the buttons on the overlays, but not the box text (this would be necessary)
    b. the cicerone guide can only be accessed after reading through the entirety of the page first, then it wraps through cicerone - can cicerone get a priority tag somehow? (this may be lower priority, but would be helpful)
3. a screen reader is not reading the "more info" button at the bottom right of the page
